testnets/titantestnet/assetlist.json

```diff
@@ -199,6 +199,162 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/eth.svg"
         }
       ]
+    },
+    {
+      "description": "USDC stablecoin on Titan. Bridge into titan chain from Ethereum to Titan through Titan Power flow bridge https://dev.powerflow.asia/bridge",
+      "denom_units": [
+        {
+          "denom": "factory/titan16gyvmp43st00s220zypex4lgvwqc3ve8l4657rhxj8myeadswmkq75slkc/usdc",
+          "exponent": 0,
+          "aliases": [
+            "uusdc"
+          ]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "factory/titan16gyvmp43st00s220zypex4lgvwqc3ve8l4657rhxj8myeadswmkq75slkc/usdc",
+      "name": "USDC",
+      "display": "usdc",
+      "symbol": "USDC",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereumtestnet",
+            "base_denom": "0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238",
+            "contract": "0x63e4dAcAFbBf0a02d70e5b47506eebEDDe49b1c6"
+          },
+          "provider": "powerflow.asia, titanlab.io"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
+        }
+      ]
+    },
+    {
+      "description": "SOL on Titan. Bridge into titan chain from Solana devnet to Titan through Titan Power flow bridge https://dev.powerflow.asia/bridge",
+      "denom_units": [
+        {
+          "denom": "factory/titan16gyvmp43st00s220zypex4lgvwqc3ve8l4657rhxj8myeadswmkq75slkc/sol",
+          "exponent": 0,
+          "aliases": [
+            "lamport"
+          ]
+        },
+        {
+          "denom": "sol",
+          "exponent": 9
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "factory/titan16gyvmp43st00s220zypex4lgvwqc3ve8l4657rhxj8myeadswmkq75slkc/sol",
+      "name": "SOL",
+      "display": "sol",
+      "symbol": "SOL",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "solanatestnet",
+            "base_denom": "So11111111111111111111111111111111111111112",
+            "contract": "EYREJzRmAXGFTZh1hhH4gtBr7mM7gqSY5bzPYszL2XtD"
+          },
+          "provider": "powerflow.asia, titanlab.io"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/titantestnet/images/sol.png"
```

Comment:
Instead of uploading a duplicate image for each asset, please just keep one copy of the image in the (mainnet) /titan/images/ directory, and refer to that image from here.